docs/tasks/Base.md

```diff
@@ -56,7 +56,9 @@ $this->taskExecStack()
 * `stopOnFail($stopOnFail = null)`   * `param bool` $stopOnFail
 * `result($result)` 
 * `dir($dir)`  Changes working directory of command
-* `printed($arg)`  Should command output be printed
```

Comment:
This was removed from docs but is still present in code to prevent a backwards incompatible change.